src/rgw/rgw_sal_dbstore.cc

```diff
@@ -1660,7 +1660,7 @@ namespace rgw::sal {
   int DBStore::get_user_by_swift(const DoutPrefixProvider *dpp, const std::string& user_str, optional_yield y, std::unique_ptr<User>* user)
   {
     /* Swift keys and subusers are not supported for now */
-    return 0;
+    return -1;
```

Comment:
We decided in the other PR that these should return -ENOTSUP

Reply:
Done.